web/src/shared/components/DarkModeTriButton/DarkModeTriButton.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+import './DarkModeTriButton.css'
+import LightModeIcon from '@mui/icons-material/LightMode'
+import DarkModeIcon from '@mui/icons-material/DarkMode'
+import ComputerIcon from '@mui/icons-material/Computer'
+import { Popup } from 'semantic-ui-react'
+import { ThemeContext } from '../ThemeProvider'
+
+const DarkModeTriButton: React.FunctionComponent = () => {
+    const { toggleTheme } = React.useContext(ThemeContext)
+    const [selection, updateSelection] = React.useState(
+        localStorage.getItem('theme') || 'system-mode'
+    )
+    return (
+        <div className="switch-toggle">
+            <input
+                id="on"
+                name="state-d"
+                type="radio"
+                onChange={() => {
+                    toggleTheme('light-mode')
```

Comment:
Might be reasonable to make these constants that can be referred to throughout the code to avoid mistypes

Reply:
Done as part of the loop